finance-api/api-reference.md

```diff
@@ -0,0 +1,282 @@
+
+<details open="true">
+<summary>Parameters</summary>
+
+|Name |Type |In |Required/Optional |Description
+|:---- |:---- |:---- |:---- |:----
+|organizationUuid |string |path |required |Unique identifier for your organization. You can specify the value with one of the following: <br/><ul><li> Use `self` as the value. This will retrieve your organizationUuid from the authentication token in the request.</li><li> Get it by using the https://oauth.izettle.com/users/me endpoint of OAuth2 API. See [OAuth2 API](https://github.com/iZettle/api-documentation/blob/master/authorization.adoc) for more information.</li></ul> 
+|at |string |query |optional |Used to fetch payouts until a time in the UTC time zone. If it's used, any transaction after that point will be ignored. If it's not used, the balance of all transactions at the current point of time is returned. <br/>You can specify a UTC time in one of the following formats: <br/><ul><li>`YYYY-MM-DD` to specify a date. For example, `2020-11-29`.</li><li>`YYYY-MM-DDThh:mm:ss` to specify a time. For example, `2020-11-29T03:10:02`.</li></ul>
+</details>
+
+
+### Responses
+<details open="true">
+<summary>HTTP status codes</summary>
+
+|Status code |Description
+|:---- |:----
+|200 OK |Returned when the payout information is successfully fetched.  
+|403 Forbidden |Returned when you are not authorised to fetch the payout information for the account.
+|400 Bad Request |Returned when a required parameter is missing or in a wrong format in the request.  
+</details>
+
+<details open="true">
+<summary>Response attributes</summary>
+
+|Name |Type |Description
+|:---- |:---- |:----
+|data |object|Information about the account payout 
+|totalBalance |integer |The account balance. It can be negative. For example, `-300`.
+|currencyId |string |The currency of the account balance. For example, `SEK`. 
+|nextPayoutAmount |integer |The amount of money that is in the `liquid` status and is ready to be paid out to the merchant.  
+|discountRemaining |integer |The amount of discounts that remains in merchant's vouchers.  
+|periodicity |string |The interval between each payout. It can be `DAILY`, `WEEKLY`, and `MONTHLY`.   
+</details>
+
+
+## Examples
+
+### Fetch balance for a liquid account
+The following example fetches balance from Zettle liquid account of the merchant that has `organizationUuid` as `18071066-bfc7-11eb-8529-0242ac130003`. As `at` is not specified for a time point, by default, it fetches the balance that is available at the moment of fecthing.
+
+Request
+```
+GET/organizations/18071066-bfc7-11eb-8529-0242ac130003/accounts/liquid/balance
+```
+Response
+```json
+{
+    "data": {
+        "totalBalance": 195,
+        "currencyId": "GBP"
+    }
+}
+```
+
+### Fetch transactions for a liquid account
+The following example fetches all transactions from Zettle liquid account of the merchant that has `organizationUuid` as `18071066-bfc7-11eb-8529-0242ac130003` from 1 January, 2021 to 7 June, 2021.
```

Comment:
> fetches all transactions
all types of transactions

Reply:
Why use orgUuid in the example and not `self` alias?
I think using orgUuid makes it more complex and hard to digest. I would rather have an example using alias since the orgUuid is present in the oauth token already.